.gitignore

```diff
@@ -13,3 +13,4 @@ toolkit/*/work*
 local.mk
 *.DS_Store
 *~
+.log
```

Comment:
please undo to avoid merge conflicts 
we already have entries to ignore the created log files by package builds (build*, publish*)
and there is a package that includes a log file (spk/aria2/src/aria2.log - but IMHO this could be reworked to be obsolete)
```suggestion
```

Reply:
Thanks, I'll update it now